client/scripts/directives/previewPictures.js

```diff
@@ -0,0 +1,26 @@
+require('../app').directive('previewPictures', /* @ngInject */function () {
+  return {
+    scope: {
+      pictures: '=previewPictures'
+    },
+    require: 'previewPictures',
+    bindToController: true,
+    link: function ($scope, $element, $attrs, $ctrl) {
+      $element.on('click', $ctrl.preview)
+    },
+    controller: /* @ngInject */function ($attrs, $filter, Lightbox) {
```

Comment:
No need for unused dependencies
```suggestion
    controller: /* @ngInject */function ($filter, Lightbox) {
```